content/_includes/birds.macros.njk

```diff
@@ -94,7 +94,7 @@ params:
     {%- set name = all or bird -%}
     {{ utility.link_if(
       content=name | mdInline | safe,
-      url=page.url if link else none,
+      url=page.page.url if link else none,
```

Comment:
11ty [recommends](https://www.11ty.dev/docs/collections/#collection-item-data-structure) accessing these props off the `page` object when in the context of a collection.